docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-feature-integration.md

```diff
@@ -17,18 +17,17 @@ To start feature integration, integrate the required features:
 
 | NAME | VERSION | INTEGRATION GUIDE |
 | --------- | ------ | ---------------|
-| Spryker Core | 202001.0 | [Spryker Core feature integration](https://documentation.spryker.com/docs/spryker-core-feature-integration) |
-| Order Management | 202001.0 | [Order Management feature integration](https://documentation.spryker.com/docs/order-management-feature-integration) |
-| Shipment | 202001.0 | [Shipment feature integration](https://documentation.spryker.com/docs/shipment-feature-integration) |
-| State Machine | 202001.0 | [State Machine feature integration](https://github.com/spryker-feature/state-machine) |
+| Order Management | {{page.version}} | [Order Management feature integration](https://documentation.spryker.com/docs/order-management-feature-integration) |
+| State Machine | {{page.version}} | [State Machine feature integration](https://github.com/spryker-feature/state-machine) |
 | Marketplace Merchant | {{page.version}} | [Marketplace Merchant feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-merchant-feature-integration.html) |
+| Marketplace Shipment | {{page.version}} | [Marketplace Shipment feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-shipment-feature-integration.html) |
 
 ### 1) Install required modules using Сomposer
 
 Install the required modules:
 
 ```bash
-composer require spryker-feature/marketplace-order-management --update-with-dependencies
+composer require spryker-feature/marketplace-order-management:"dev-master" --update-with-dependencies
```

Comment:
please use release tag

Reply:
commands for features should be with `dev-master`